Src/VimCore/FSharpUtil.fs

```diff
@@ -580,6 +580,7 @@ module CharUtil =
     let IsLower x = System.Char.IsLower(x)
     let IsLowerLetter x = IsLower x && IsLetter x
     let IsLetterOrDigit x = System.Char.IsLetterOrDigit(x)
+    let IsIdent x = IsLetterOrDigit x || x = '_'

```

Comment:
This should actually be the content of `isident` setting vs. a hard coded list. Can you file a bug to track implementing this and moving this function over? It should actually be pretty easy once I complete the `iskeyword` feature. The `isindent` option is simple to build on top of that support.

Reply:
Great idea to file as issue so I don't forget.

Reply:
Opened #2645.